eclair-core/src/main/resources/reference.conf

```diff
@@ -224,6 +224,7 @@ eclair {
       username = ""
       password = ""
       readonly-user = "" // if defined, this user will be granted read-only access to all tables in the database
+      reset-json-columns = false // in case of a json format change, this allows a full re-serialization of json data
```

Comment:
> I think people will almost always forget to set this to true when needed.

They may, but I don't think it's a big deal.

I believe we should make no guarantee whatsoever on json backwards compatibility, and accept that this column is here for analytics/advanced admin purposes. Otherwise very change to the codecs will be hard. In my view, we shouldn't even bother with that and only trigger a json reset when some SQL query starts to fail.

NB: the full json rewrite is fast, it's only a few 10k lines, even on our large node.